pkg/3scale/amp/component/apicast_monitoring.go

```diff
@@ -151,6 +151,18 @@ func ApicastPrometheusRules(ns string) *monitoringv1.PrometheusRule {
 								"severity": "warning",
 							},
 						},
+						{
+							Alert: "ThreescaleApicastWorkerRestart",
+							Annotations: map[string]string{
+								"summary":     "A worker process in Nginx has been started",
+								"description": "A new thread has been started that means that maybe any worker process died because of any memory limit. Please investigate the memory pressure on pod (instance {{ $labels.instance }})",
+							},
+							Expr: intstr.FromString(fmt.Sprintf(`changes(worker_process{namespace='%s', pod=~'apicast-production.*'}[5m]) > 0`, ns)),
+							For:  "5m",
+							Labels: map[string]string{
+								"severity": "warning",
```

Comment:
@obrienrobert please confirm that warning is ok to you, I think that it's ok, but please confirm.

Reply:
@eloycoto I know there's work on SOP's currently being carried out by the 3scale team ([THREESCALE-6117](https://issues.redhat.com/browse/THREESCALE-6117)) Is there a SOP being created for this alert? I believe that this alert should be critical due to the nature of the problem, and we always try to include a SOP for critical alerts to allow the alert to be action by SRE.

Reply:
yep, SOP will land next week ;-)

Reply:
In my opinion Critical alerting should be very justified and have actionable steps and be fixable once the alert happens.

Are those conditions met?

Reply:
cc @obrienrobert @eloycoto

Reply:
> Critical alerting should be very justified

I agree, and it's one of the reasons why a SOP is being created for this issue, so it can be actioned. 

> have actionable steps and be fixable once the alert happens

This is where the SOP comes into play

Reply:
@pepedocs FYI

Reply:
@eloycoto can you confirm the alert/sop will be actionable with fixable steps? I'm aware there will be a SOP, but is the SOP going to contain actionable steps / steps that fix the problem when the alert is triggered, and that are compatible with the environment/conditions where 3scale installation is going to be run?

Reply:
can confirm yes.

Reply:
:+1: thanks for your answer.

I've updated the alert severity to `critical`